app/src/main/java/eu/kanade/tachiyomi/data/backup/restore/BackupRestorer.kt

```diff
@@ -157,6 +166,19 @@ class BackupRestorer(
         )
     }
 
+    private fun CoroutineScope.restoreExtensionRepos(backupExtensionRepo: List<BackupExtensionRepos>) = launch {
+        ensureActive()
+        extensionRepoRestorer(backupExtensionRepo)
+
+        restoreProgress += 1
+        notifier.showRestoreProgress(
+            context.stringResource(MR.strings.categories),
```

Comment:
?

Reply:
🤣I was reusing code from the `Categories` backup and restore. I guess I forgot to switch `context.stringResource(MR.strings.categories),`